go.mod

```diff
@@ -20,114 +20,195 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/spf13/cobra v1.3.0
 	github.com/stretchr/testify v1.7.0
-	github.com/talos-systems/capi-utils v0.0.0-20211126110629-e8c3bf93e75f
+	github.com/talos-systems/capi-utils v0.0.0-20220317162331-3a6f8ee7b753
 	github.com/talos-systems/go-retry v0.3.1
 	github.com/talos-systems/grpc-proxy v0.2.0
 	github.com/talos-systems/talos v0.15.0-alpha.1.0.20220127120728-831f65a07f3b
```

Comment:
should we upgrade Talos as well? use latest beta?

Reply:
sure, let's do it.